### PR TITLE
Add link to Privacy Policy in onboarding

### PR DESCRIPTION
**Pull request by andresbase**
The onboarding is missing the link to our Privacy Policy. Since this is what the user sees when clicking the question mark, it seems the right place to add a link to it.

### CODE REVIEW

File: src/_locales/en_US/messages.json [hunk 1]
```diff
@@ -43,6 +43,10 @@
         "message": "Help us by donating and sharing your support for our tools",
         "description": "Shown below the Donate button on the intro page."
     },
+    "intro_link_policy": {
+        "message": "<a href='https://www.eff.org/code/privacy/policy' target='_blank'>Privacy Policy</a>",
```
Comment (ghostwords) [inline src/_locales/en_US/messages.json]: We should move the anchor link out of the translation message into the HTML document. No need to complicate translations in this case, I think.

File: src/_locales/en_US/messages.json [hunk 2]
```diff
@@ -43,6 +43,10 @@
...
+        "description": "Link to EFF's software projects Privacy Policy."
```
Comment (ghostwords) [inline src/_locales/en_US/messages.json]: Should mention where this link can be found (on the bottom of the intro page).

File: src/skin/css/firstRun.css [hunk 3]
```diff
@@ -788,6 +788,12 @@ body {
   background-image:linear-gradient(#DFDFE6, #DFDFE6 30%, #fff 30%);
   padding:4rem 0;
 }
+#pb-privacy-policy {
+  color:#F06A0A;
+  font-family:"OpenSans",sans-serif;
```
Comment (ghostwords) [inline src/skin/css/firstRun.css]: We ship Privacy Badger with OpenSans light and bold fonts only ... Not sure if this works correctly.

**Comment by andresbase** [comment]
With the changes:
privpolpb

File: src/skin/css/firstRun.css [hunk 4]
```diff
@@ -788,6 +788,13 @@ body {
   background-image:linear-gradient(#DFDFE6, #DFDFE6 30%, #fff 30%);
   padding:4rem 0;
 }
+#pb-privacy-policy, #pb-privacy-policy a {
+  color:#707070;
+  font-family:"OpenSans-Light",sans-serif;
+  font-size:0.7rem;
+  text-align:center;
+  padding-bottom: 0rem;
```
Comment (ghostwords) [inline src/skin/css/firstRun.css]: Is the `padding-bottom` necessary? I didn't notice a difference without it.

File: src/skin/css/firstRun.css [hunk 5]
```diff
@@ -788,6 +788,13 @@ body {
   background-image:linear-gradient(#DFDFE6, #DFDFE6 30%, #fff 30%);
   padding:4rem 0;
 }
+#pb-privacy-policy, #pb-privacy-policy a {
+  color:#707070;
+  font-family:"OpenSans-Light",sans-serif;
+  font-size:0.7rem;
```
Comment (ghostwords) [inline src/skin/css/firstRun.css]: Maybe make the text a bit larger?

File: src/skin/firstRun.html [hunk 6]
```diff
@@ -124,5 +124,13 @@ <h3 class="i18n_intro_donate_heading"></h3>
     </a>
   </div>
 
+  <!-- Privacy Policy -->
+  <div class="grid-container full">
+    <div id="pb-privacy-policy" class"align-center">
```
Comment (ghostwords) [inline src/skin/firstRun.html]: Typo: missing "=".

File: src/skin/css/firstRun.css [hunk 7]
```diff
@@ -788,6 +788,13 @@ body {
...
+  text-align:center;
```
Comment (ghostwords) [inline src/skin/css/firstRun.css]: May not be needed once HTML typo is fixed.

File: src/skin/firstRun.html [hunk 8]
```diff
@@ -124,5 +124,13 @@ <h3 class="i18n_intro_donate_heading"></h3>
...
+      <!-- <span class="i18n_intro_link_policy"></span> -->
```
Comment (ghostwords) [inline src/skin/firstRun.html]: Should remove.

File: src/_locales/en_US/messages.json [hunk 9]
```diff
@@ -43,6 +43,10 @@
         "message": "Help us by donating and sharing your support for our tools",
         "description": "Shown below the Donate button on the intro page."
     },
+    "intro_link_policy": {
+        "message": "Privacy Policy",
+        "description": "Shown at the bottom of the intro page, links to Privacy Policy."
```
Comment (ghostwords) [inline src/_locales/en_US/messages.json]: "... links to the EFF software and technology projects privacy policy"?

File: src/skin/css/firstRun.css [hunk 10]
```diff
@@ -788,6 +788,13 @@ body {
   background-image:linear-gradient(#DFDFE6, #DFDFE6 30%, #fff 30%);
   padding:4rem 0;
 }
+#pb-privacy-policy, #pb-privacy-policy a {
```
Comment (ghostwords) [inline src/skin/css/firstRun.css]: Do we need the first selector?

Comment (andresbase) [reply src/skin/css/firstRun.css]: To center the div for the grid.

File: src/skin/css/firstRun.css (hunk 7 above)
Comment (andresbase) [reply src/skin/css/firstRun.css]: I ended up removing the class as it wasn't doing anything and this was still needed.

File: src/skin/css/firstRun.css (hunk 5 above)
Comment (andresbase) [reply src/skin/css/firstRun.css]: I tried with 0.9, 1 seems a bit too large. Let me know how this looks.

**Comment by ghostwords** [comment]
Followup commits: https://github.com/EFForg/privacybadger/compare/5555122fb7b4cfe030a461566b4670662dff310e%5E...334cf44b44235eacb4a1ff708914d8469cae59d1.

**Pull request merged**